src/mappings/gns.ts

```diff
@@ -72,10 +73,26 @@ export function handleSetDefaultName(event: SetDefaultName): void {
       indexer.defaultDisplayName = graphAccount.defaultDisplayName
       indexer.save()
     }
+    addDefaultNameTokenLockWallets(graphAccount, newDefaultName)
   }
   graphAccount.save()
 }
 
+// Add in default names to a graph accounts token lock wallets
+function addDefaultNameTokenLockWallets(graphAccount: GraphAccount, name: string): void {
+  let tlws = graphAccount.tokenLockWallets
+  for (let i = 0; i < tlws.length; i++) {
+    let tlw = GraphAccount.load(tlws[i])
```

Comment:
Are we sure this doesn't return null?
When we create the TLWs in `handleTokenLockCreated` we aren't creating a GraphAccount for it (only for its beneficiary), although it can create one by interacting with the protocol at any point (for example, staking to be an indexer), but what happens with those that don't interact at all? 🤔

Reply:
Mostly asking because, since we don't check for it to be null, we can make the subgraph fail on `.save()`, and we could easily fix it by checking for null. Although I might be missing something and it could be initializing the `GraphAccount` every time 😅

Reply:
I think we should create a graph account for the token lock wallet in `handleTokenLockCreated` . I actually thought that was what I did, but I realize that I only did the beneficiary. 

So if we create both in there, then I believe that should satisfy the situation. What do you think? 

(Also we will still confirm when it is synced)